facia/app/controllers/FaciaController.scala

```diff
@@ -148,13 +149,17 @@ trait FaciaController extends Controller with Logging with ExecutionContexts wit
     val futureResult = frontJsonFapi.get(path).flatMap {
       case Some(faciaPage) =>
         successful(
-          Cached(faciaPage) {
-            if (request.isRss)
-              Ok(TrailsToRss.fromPressedPage(faciaPage)).as("text/xml; charset=utf-8")
+          Cached.withEtag(faciaPage) {
+            if (request.isRss) {
+              val body = TrailsToRss.fromPressedPage(faciaPage)
+              StringResult(Ok(body).as("text/xml; charset=utf-8"), body)
+            }
             else if (request.isJson)
-              JsonFront(faciaPage)
-            else
-              Ok(views.html.front(faciaPage))
+              StringResult.withoutString(JsonFront(faciaPage))
```

Comment:
`StringResult.withoutString` seems confusing. Could we call it something more intuitive?